src/sentry/notifications/notifications/base.py

```diff
@@ -126,8 +126,15 @@ def get_message_actions(self, recipient: Team | User) -> Sequence[MessageAction]
     def get_callback_data(self) -> Mapping[str, Any] | None:
         return None
 
-    def record_analytics(self, event_name: str, **kwargs: Any) -> None:
-        analytics.record(event_name, **kwargs)
+    @property
+    def analytics_instance(self) -> Any | None:
+        """
+        Returns on instance for that can be used for analytics such as an organization or project
```

Comment:
Do you mean:

"Returns an instance that can be used ... " ?

Reply:
@ceorourke LOL yes